src/main/java/com/theelixrlabs/healthcare/constants/MessageConstants.java

```diff
@@ -0,0 +1,7 @@
+package com.theelixrlabs.healthcare.constants;
+
+public class MessageConstants {
+    public static final String DB_COLLECTION_NAME = "doctor_patient_assignment";
```

Comment:
why in message constants ?
There should be different constant file for specific to these fields 
do similar to DoctorsConstants,java

Reply:
Resolved.